cni/netconfig.go

```diff
@@ -80,7 +80,9 @@ type NetworkConfig struct {
 }
 
 type WindowsSettings struct {
-	EnableLoopbackDSR bool `json:"enableLoopbackDSR,omitempty"`
+	EnableLoopbackDSR         bool `json:"enableLoopbackDSR,omitempty"`
+	EnableHNSTimeout          bool `json:"enableHNSTimeout,omitempty"`
+	HnsTimeoutDurationSeconds int  `json:"hnsTimeoutDurationSeconds,omitempty"`
```

Comment:
make this a `time.Duration` so we can't use it incorrectly

Reply:
can't be unmarshalled directly but check [`time.ParseDuration`](https://pkg.go.dev/time#ParseDuration)

Reply:
Yeah, can't be unmarshalled directly. I'll keep it as an integer, easier to glean the format from the conflist and also easier to do the conditional check.